front/app/hooks/useHomepageSettingsFeatureFlag.ts

```diff
@@ -40,7 +40,7 @@ export default function useHomepageSettingsFeatureFlag({
       ? appConfig.data.attributes.settings?.[homePageAllowedSettingName]
       : null;
   const isEnabled = !isNilOrError(homepageSettings)
-    ? homepageSettings.attributes[homepageEnabledSetting]
+    ? homepageSettings.data.attributes[homepageEnabledSetting]
```

Comment:
We usually have a Data type then, as you can see in most services files (eg IIdeaData in ideas.ts). We can also return the content of data, so we don’t need to do the .data each time (eg useIdea.ts).

Reply:
+1, though this change will break your branch (I think) and my work on 1010 + 1011 so it might be worth doing later as a refactor